dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@paw%2Fmetadata-freshness#egg=dbt-core&subdirectory=core
```

Comment:
should we change `%2` to a `/`? don't recall having to make this swap

Reply:
You mean you don't manually type it in and you copy/pasta from the URL? Blasphemy!